itou/templates/apply/includes/job_application_answers.html

```diff
@@ -21,4 +22,57 @@ <h4 class="h6 mt-3">Message envoyé au candidat</h4>
             {{ job_application.answer|linebreaks }}
         </blockquote>
     {% endif %}
+
+    {% if display_refusal_info %}
+        <div class="c-info">
+            <button class="c-info__summary collapsed"
+                    {% matomo_event "candidature" "clic" "prescriber_displayed_company_details" %}
+                    data-bs-toggle="collapse"
+                    data-bs-target="#collapseInfoExample"
+                    aria-expanded="false"
+                    aria-controls="collapseInfoExample">
+                <span>Contactez l’employeur pour en savoir plus</span>
+            </button>
+            <div class="c-info__detail collapse" id="collapseInfoExample">
+                <p>
+                    L’employeur a refusé la candidature avec le motif “Autre”.
+                    <br>
+                    Si les détails apportés dans le message de réponse ne vous ont pas permis d’en savoir plus, vous pouvez contacter l’employeur.
+                </p>
+
+                <p class="text-primary fs-sm font-weight-bold my-3">Coordonnées de l’employeur :</p>
+                <ul class="list-unstyled mb-lg-5">
+                    {% if refused_by %}
+                        <li class="d-flex justify-content-start align-items-center">
+                            <a href="{% url 'companies_views:card' siae_id=job_application.to_company.pk %}" class="btn-link btn-ico">
+                                <i class="ri-user-line ri-lg font-weight-normal"></i>
+                                <span>{{ refused_by.get_full_name }}</span>
+                            </a>
+                        </li>
+                    {% endif %}
+                    <li class="d-flex justify-content-start align-items-center">
+                        <a href="{% url 'companies_views:card' siae_id=job_application.to_company.pk %}" class="btn-link btn-ico">
+                            <i class="ri-community-line ri-lg font-weight-normal"></i>
+                            <span>{{ job_application.to_company.display_name }}</span>
+                        </a>
+                    </li>
+                    <li class="d-flex justify-content-start align-items-center">
+                        <a href="mailto:{{ refusal_contact_email }}" class="btn-link btn-ico">
+                            <i class="ri-mail-line ri-lg font-weight-normal"></i>
+                            <span>{{ refusal_contact_email }}</span>
+                        </a>
+                        {% include "includes/copy_to_clipboard.html" with text="‎" content=refusal_contact_email css_classes="btn-link fs-sm py-0" %}
+                    </li>
+                    <li class="d-flex justify-content-start align-items-center">
+                        <a href="tel:{{ job_application.to_company.phone }}" class="btn-link btn-ico">
```

Comment:
```suggestion
                        <a href="tel:{{ job_application.to_company.phone|cut:" " }}" class="btn-link btn-ico">
```
Les href ne sont pas censés contenir d'espace (et nos numéros de téléphone en ont parfois)